crates/core/tedge_agent/src/software_manager/config.rs

```diff
@@ -11,6 +11,7 @@ pub struct SoftwareManagerConfig {
     pub log_dir: Utf8PathBuf,
     pub default_plugin_type: Option<String>,
     pub config_location: TEdgeConfigLocation,
+    pub sudo_command_builder: SudoCommandBuilder,
```

Comment:
[minor] Here we have a 3rd name for the same thing.

```suggestion
    pub sudo: SudoCommandBuilder,
```